static/third/bootstrap/css/bootstrap.css

```diff
@@ -1470,7 +1470,7 @@ input[readonly],
 select[readonly],
 textarea[readonly] {
   cursor: not-allowed;
-  background-color: #eeeeee;
+  background-color: #ffffff;
```

Comment:
We strongly don't recommend changing css into `bootstrap.css` file. Because this will affect every element in chat.

Reply:
yes I feel the same but I tried different methods for fixing this issue but either the simple version got disturbed or either the night version the only solution found in which both of them worked was after changing the bootstrap file.Can you provide some hints for solving this issue in other way?

Reply:
Maybe we can remove the `backgroud-color` of input field and make it transparent? So that background color of chat will be the background color of input element as well.

Reply:
Sorry @YJDave for late reply was busy with practicals yes i think this would be better i can try ang get back to you soon

Reply:
Sorry for a late reply @YJDave i forgot about this issue.Can you please till i have to remove background  from bootstrap or  i  have to redefine the color in css file?

Reply:
Posted a quick comment on one of the issues.